test/algorithms/Jamfile

```diff
@@ -39,9 +39,6 @@ test-suite boost-geometry-algorithms
     [ run is_valid.cpp                 : : : : algorithms_is_valid ]
     [ run is_valid_failure.cpp         : : : : algorithms_is_valid_failure ]
     [ run is_valid_geo.cpp             : : : : algorithms_is_valid_geo ]
-    [ run is_valid.cpp                 : : : <define>BOOST_GEOMETRY_ROBUSTNESS_ALTERNATIVE : algorithms_is_valid_alternative ]
-    [ run is_valid_failure.cpp         : : : <define>BOOST_GEOMETRY_ROBUSTNESS_ALTERNATIVE : algorithms_is_valid_failure_alternative ]
-    [ run is_valid_geo.cpp             : : : <define>BOOST_GEOMETRY_ROBUSTNESS_ALTERNATIVE : algorithms_is_valid_geo_alternative ]
     [ run line_interpolate.cpp         : : : : algorithms_line_interpolate ]
```

Comment:
In general, I think we should remove these in a separate PR.
Now it looks just like it's done because the change still gives errors in rescaling code.
By chance, I added all these in the cmake PR - but if it is already removed, that part is neither necessary there anymore (or it will break after merging this PR).

Can you please split it?

Reply:
I need this is this PR because there tests are failing with the new fix. I will place them in a different commit.